### DISCUSSION TITLE
[LTS]Tenants - 2850306027

### DISCUSSION THREAD

**Discussion by zilrty** [RimWorldKorea/RMK#706] [일반 - 림추출기]
모드 :  https://steamcommunity.com/sharedfiles/filedetails/?id=2850306027
번역 xlsx : [[LTS]Tenants - 2850306027.xlsx](https://github.com/RimWorldKorea/RMK/files/15037549/LTS.Tenants.-.2850306027.xlsx)

업뎃 작업 중인데

LTS_Tenancy.questNameRules.rulesStrings.0
LTS_Tenancy.questNameRules.rulesStrings.1
LTS_Tenancy.questNameRules.rulesStrings.2
LTS_Tenancy.questNameRules.rulesStrings.3

얘네들이 xml로 변환할때 변환이 안되는거 같음

다른 rulesStrings 부분은 변환이 잘되는데 쟤네만 자꾸 누락되서

일단은 그냥 xml 파일에 직접 적어논 상태

**Comment by RimWorldKorea** [comment]
https://github.com/csh1668/RimworldExtractor/issues/6
